extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceTest.php

```diff
@@ -10,49 +10,17 @@ protected function setUp() {
 		parent::setUp();
 	}
 
-	/**
-	 * @param $input to check presence of some additional config fields. Possible fields:
-	 * 'isInvalidImage' - bool - if getThumbnail should return false
-	 * 'isMobile' - bool - if we want to test mobile env
-	 * 'isMercury' - bool - if we want to test Mercury skin
-	 * 'isMercuryExperimentalMarkupEnabled' - bool
-	 * 'smallImageDimensions' - integer - size of small image (both width and height)
-	 *
-	 * @return PHPUnit_Framework_MockObject_MockObject
-	 */
 	private function mockInfoboxRenderServiceHelper( $input ) {
-		$isValidHeroDataItem = isset( $input[ 'isValidHeroDataItem' ] ) && $input[ 'isValidHeroDataItem' ];
-		$isMobile = isset( $input[ 'isMobile' ] ) && $input[ 'isMobile' ];
-		$isMercury = isset( $input[ 'isMercury' ] ) && $input[ 'isMercury' ];
+		$extendImageData = isset( $input['extendImageData'] ) ? $input['extendImageData'] : null;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "extendImageData" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)